DorisProject/src/app/pages/modal/modal.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, Inject, OnInit} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
```

Comment:
Codacy has a fix for the issue: [Replace `MAT_DIALOG_DATA,·MatDialogRef` with `·MAT_DIALOG_DATA,·MatDialogRef·`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
```